### DISCUSSION TITLE
Theme developing

### DISCUSSION THREAD

**Discussion by abdosaeedelhassan** [statamic/cms#6462] [Q&A]
Hi,
is there any video tutorial how to develop starter kit theme from scratch, i already read the documentation but it not understandable for me.
Thank you

**Comment by duncanmcclean** [comment]
Hello 👋

Essentially, you build a site like normal, then when you're done you follow the [starter kit documentation](https://statamic.dev/starter-kits/creating-a-starter-kit#exporting-files) to export it for others to use. 

Is there anything in particular you're struggling to understand?

**Reply by abdosaeedelhassan** [reply]
Welcome  @duncanmcclean 
1- i did the the structure of files as (https://github.com/statamic/starter-kit-starters-creek) inside packages folder in statamic project.
2- added the path of the package folder in composer.json file in main project ( statamic project).
3- i did the command (php please starter-kit:export package path), this command create me the (starter-kit.yaml) file but in main project then i moved inside my theme folder as found in step 1
4- finally i did php please starter-kit:install package/name, and it clean the statamic project and i think installed by package, but not worked, no home page of blogs folder and other

i think if there a video tutorial from scratch contain all step with simple theme will be  very helpful.
Thank you
